src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * AddCommand class that encompasses the todo, deadline, event commands
+ */
+public class AddCommand extends Command {
+    protected String fullCommand;
+    protected String[] s;
+
+    /**
+     * Creates a new AddCommand object that has takes note of the full command and the command separated by a space
+     * @param fullCommand input by user
+     * @param s parsed command with spaces removed
+     */
+    public AddCommand(String fullCommand, String[] s) {
+        this.fullCommand = fullCommand;
+        this.s = s;
+    }
+
+    /**
+     * A method that performs the desired action of the command
```

Comment:
Clean and concise documentation that is easy to understand. Perhaps can be made neater by adding a line between the method description and params?